### PR TITLE
Bugfix for issue #1449 (https://github.com/busyorg/busy/issues/1499) …

### LINKED ISSUE #1499: Possible minor mistakes in webpack configuration
https://github.com/busyorg/busy/blob/bd5245616e1bd63eec31960ce5bb1dcd8aa630b7/webpack/makeConfig.js#L119
https://github.com/busyorg/busy/blob/bd5245616e1bd63eec31960ce5bb1dcd8aa630b7/webpack/makeConfig.js#L142
https://github.com/busyorg/busy/blob/bd5245616e1bd63eec31960ce5bb1dcd8aa630b7/webpack/makeConfig.js#L191

**Lines 119, 142**
I think what is meant to be in the regexp instead of `/\.css|.less$/`: `/(\.css|\.less)$/i`

**Line 191**
Replace `/\.js?$/` with this `/\.jsx?$/i`

*(lines 119, 142)*
 - **Current** - https://regexper.com/#%2F%5C.css%7C.less%24%2F
 - **Fixed** - https://regexper.com/#%2F(%5C.css%7C%5C.less)%24%2Fi

*For reference (line 191):*
 - **Current** - https://regexper.com/#%2F%5C.js%3F%24%2F
 - **Fixed** - https://regexper.com/#%2F%5C.jsx%3F%24%2F

### PR DESCRIPTION
**Pull request by vsnikkil**
Possible minor mistakes in webpack configuration

Fixes #1499

Changes:
- webpack/

### CODE REVIEW

File: webpack/configUtils.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/**
```
Comment (bonustrack) [inline webpack/configUtils.js]: I don't think we need this file, IMO it's an unnecessary complexity.

Comment (vsnikkil) [reply webpack/configUtils.js]: I also agree, I fill fix it as soon as I find a way to share same matchers between files

Comment (Sekhmet) [reply webpack/configUtils.js]: I think adding this helper function adds too much complexity.
I would either leave them hardcoded or just add constants for it.

File: webpack/configUtils.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * Create a regular expression matcher for filename extensions.
+ */
+const createFnExtMatcher = (...fileExtensions) => {
+  if (typeof fileExtensions !== 'object' || !Array.isArray(fileExtensions)) {
+    throw new Error('`fileExtensions` is not an array');
+  }
+
+  const matcher = new RegExp(`\.(${ fileExtensions.join('|') })$`, 'i');
```
Comment (vsnikkil) [inline webpack/configUtils.js]: `fileExtensions` items should be escaped (e.g., calling the function with arguments `"false|extension", js"` will result in following RegExp: `/.(false|extension|js)$/i`. That would match files with extension false, extension, and js)

**Comment by vsnikkil** [comment]
I rebased my PR with current `master`, in case you still want to apply this PR

**Comment by jm90m** [comment]
Thanks for the contribution @vsnikkil, we'll merge your PR

**Pull request merged**